aws_lambdas/twilio_webhook.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+from urllib.parse import unquote_plus
+
+import boto3
+from twilio.request_validator import RequestValidator
+
+from stopcovid.dialog.command_stream.publish import CommandPublisher
+from stopcovid.utils.logging import configure_logging
+
+configure_logging()
+
+
+def handler(event, context):
+    kinesis = boto3.client("kinesis")
+    stage = os.getenv("STAGE")
+
+    validator = RequestValidator(os.getenv("TWILIO_AUTH_TOKEN"))
+    url = f"https://{event['headers']['Host']}/{stage}{event['path']}"
+    form = {
+        split_pair[0]: unquote_plus(split_pair[1])
+        for split_pair in [kvpair.split("=") for kvpair in event["body"].split("&")]
+    }
```

Comment:
yeah, this is gross, but it gets a slightly different result than `urllib.parse_qs` - and that results in signature validation failure.